### PR TITLE
Added Shazam

### PR DESCRIPTION
**Pull request by KieranRobson**
# Proposed Changes

- Added Shazam
image

## Checklist

- [x] Tested locally
- [x] Ran `yarn ci` to test my code
- [x] Did not add any unnecessary changes
- [x] All my changes appear after other changes in each file
- [x] Included a screenshot (if adding a new button)
- [x] 🚀

### CODE REVIEW

File: public/css/brands.css [hunk 1]
```diff
@@ -451,4 +451,10 @@ button {
 .button.button-gitbucket {
   color: #000;
   background-color: #fff
+}
+
+/* Shazaom */
```
Comment (oliverhihn) [inline public/css/brands.css:456]: typo in comment iguess, shouldnt it say "Shazam"?

Comment (KieranRobson) [reply public/css/brands.css]: > typo in comment I guess, shouldnt it say "Shazam"?

Good catch!

**Comment by timothystewart6** [comment]
Thank you!

File: docker-compose.yml [hunk 2]
```diff
@@ -3,7 +3,8 @@
 version: "3.0"
 services:
   little-link:
-    image: ghcr.io/techno-tim/littlelink-server:latest
+    image: docker.io/library/links
```
Comment (timothystewart6) [inline docker-compose.yml:6]: I think this might be something you were testing?

Comment (KieranRobson) [reply docker-compose.yml]: I always forget that haha

**Pull request merged**